master/buildbot/changes/github.py

```diff
@@ -171,6 +198,24 @@ def _getStateObjectId(self):
             '%s/%s' % (self.owner, self.repo), self.db_class_name)
         defer.returnValue(result)
 
+    def _parseGitHubPR(self, pr_info):
+        def flatten(properties, base, info_dict):
+            for k, v in iteritems(info_dict):
+                name = ".".join([base, k])
+                if not any([
+                        prop.startswith(name)
+                        for prop in self.GH_PROPERTY_WHITELIST
```

Comment:
I looked into fnmatch but it didn't seem to provide a lot of flexibility if I use a preconfigured list. I'll look into making the list of properties configurable.